src/UserContext/AnonymousRequestMatcher.php

```diff
@@ -13,36 +13,62 @@
 
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestMatcherInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 
 /**
  * Matches anonymous requests using a list of identification headers.
  */
 class AnonymousRequestMatcher implements RequestMatcherInterface
 {
-    private $sessionNamePrefix;
+    /**
+     * The options configured in the constructor argument or default values.
+     *
+     * @var array
+     */
+    protected $options;
```

Comment:
why not private?

Reply:
i think we should remove the function in the bundle, as it should be doing the exact same thing as the function here. or is it doing something different?